doc/modules/impute.rst

```diff
@@ -93,24 +93,26 @@ string values or pandas categoricals when using the ``'most_frequent'`` or
 Multivariate feature imputation
 ===============================
 
-A more sophisticated approach is to use the :class:`ChainedImputer` class, which
-implements the imputation technique from MICE (Multivariate Imputation by
-Chained Equations). MICE models each feature with missing values as a function of
-other features, and uses that estimate for imputation. It does so in a round-robin
-fashion: at each step, a feature column is designated as output `y` and the other
-feature columns are treated as inputs `X`. A regressor is fit on `(X, y)` for known `y`.
-Then, the regressor is used to predict the unknown values of `y`. This is repeated
-for each feature in a chained fashion, and then is done for a number of imputation
-rounds. Here is an example snippet::
+A more sophisticated approach is to use the :class:`ChainedImputer` class, models
+each feature with missing values as a function of other features, and uses that
+estimate for imputation. It does so in an iterated round-robin fashion: at each step,
+a feature column is designated as output `y` and the other feature columns are treated
+as inputs `X`. A regressor is fit on `(X, y)` for known `y`. Then, the regressor is
+used to predict the unknown values of `y`. This is repeated for each feature in a
+chained fashion, and then is done for a number of imputation rounds. The results
+of the final imputation round are returned. Our implementation was inspired by the
+R MICE package (Multivariate Imputation by Chained Equations), but differs from
+it in setting single imputation to default instead of multiple imputation. This
```

Comment:
I find the "setting single imputation to default" still confusing. 
As AFAIK, the `ChainedImputer` itself can only do single imputation, that's not a matter of its default settings. You can of course then use it multiple times (setting the appropriate arguments for this case) to do multiple imputation.

Reply:
Yes, you're right, but by now users have come to expect that "chained equations" refers to multiple imputation, so the just combination "chained + single" would confuse quite a few.